pkg/manifests/manifests.go

```diff
@@ -78,6 +79,31 @@ const (
 	CustomResourceDefinitionManifest = "manifests/00-custom-resource-definition.yaml"
 )
 
+// Copy needed files from the manifests/ directory.  This is necessary in order
+// to embed these files here because manifests/ is a top-level directory in the
+// source tree, and go:embed does not allow paths containing "../".
+//
+//go:generate mkdir -p manifests
+//go:generate cp ../../manifests/00-namespace.yaml manifests/
+//go:generate cp ../../manifests/00-custom-resource-definition.yaml manifests/
+
+//go:embed assets manifests
+var content embed.FS
+
+// MustAsset returns the bytes for the named assert.
```

Comment:
```suggestion
// MustAsset returns the bytes for the named asset.
```

Reply:
Fixed in https://github.com/openshift/cluster-ingress-operator/compare/1a7236f10ce2c47f235967601d9f41685fb1a716..62697d237725ec70f45138087875c3e644b901f1.